test/FullRange.t.sol

```diff
@@ -102,17 +101,14 @@ contract TestFullRange is Test, Deployers, GasSnapshot {
         keyWithLiq = createPoolKey(token0, token2);
```

Comment:
where is `createPoolKey` from? we dont have that function in core 🤔

Reply:
it's defined in this file, it was here when i got here